pymatgen/io/core.py

```diff
@@ -255,11 +257,12 @@ class InputGenerator(MSONable):
     """
 
     @abc.abstractmethod
-    def get_input_set(self) -> InputSet:
+    def get_input_set(self, *args, **kwargs) -> Self:
```

Comment:
It seems there is still a unit test failure related to this @janosh

Reply:
thanks for fixing!

Reply:
It seems our work is overlapping. My fix in 7b8227420f244cee4c468a05b1e9efbee397a699 does not need a `ignore` 😃

Reply:
Half done. Half to go.

Reply:
I guess you have misunderstood... I mean 8583e39b10cc8aaeeba98f1ff880817962f2e14f seems to have broken one of the unit tests (split 6), do you have time to have a look? thanks

Reply:
yes, i'll fix it

Reply:
Appreciate that!

Reply:
Fixed in 2d5e0bb889ab7ccdfd0b7d3c5c65c8333145ed07. Seems like you got a typo in replacing the `LammpsInputSet` class name in:https://github.com/materialsproject/pymatgen/blob/0f3a938ed83fb4a831771904807cb55d39b39895/pymatgen/io/lammps/generators.py#L34-L82